docs/setup/_setup_main.md

```diff
@@ -141,6 +141,15 @@ flutterfire configure
     flutter run
     ```
 
+If you would rather use a demo project, you can start the Firebase Emulator and
```

Comment:
Maybe adding a link to the docs for the emulator would be ideal
https://firebase.google.com/docs/emulator-suite